app/root.jsx

```diff
@@ -8,6 +8,7 @@ import {
 } from "remix";
 import AlbyHeadIcon from "../public/images/alby_icon_head_icon.png";
 import OGImage from "../public/images/og_image.png";
+import Alby from "../public/images/alby.svg";
```

Comment:
lets use the head only version I think this fits better the way the icon is used.

Reply:
ok so you mean I should remove the tag where I added it in the 2 other places of code?